x-pack/plugins/secops/public/components/page/hosts/events_table/index.tsx

```diff
@@ -200,7 +200,7 @@ export const formatSafely = (path: string, data: Ecs): JSX.Element => {
     if (txt === '') {
       return getEmptyStringTag();
     } else {
-      return txt && txt.slice ? <>{txt.slice(0, 12)}</> : <>{txt}</>;
+      return txt && txt.slice ? <>{txt.slice(0, 45)}</> : <>{txt}</>;
```

Comment:
why 45??? I feel like 15 should have been the magic 4x3 + the three dots

Reply:
They could be IPV6 (39 without a port), no?

Reply:
oh yes!!! that's possible for sure. Also maybe the name should just describe that is just for IP  `formatSafelyIPs`

Reply:
Correct. Yea, I will change the name and I got my info from good old Stack Overflow:
https://stackoverflow.com/questions/166132/maximum-length-of-the-textual-representation-of-an-ipv6-address

I will add a quick unit test for it.